shoppingm8_fe/pubspec.lock

```diff
@@ -0,0 +1,453 @@
+# Generated by pub
```

Comment:
Wasn't it agreed to keep this file out of repository?
( https://dart.dev/guides/libraries/private-files )

Reply:
OK, I'll remove it... Didn't you add it to .gitignore?